README.md

```diff
@@ -4,3 +4,21 @@ TESTickle
 [![Build Status](https://travis-ci.org/MarkWithall/TESTickle.svg?branch=master)](https://travis-ci.org/MarkWithall/TESTickle)
 
 A unit testing library for C.
+
+Installation
+------------
+
+### Windows
+
+Download and install [Strawberry Perl](http://strawberryperl.com); this comes with `bcc` and `gmake` too. You will need to install the required CPAN libraries with: `cpan Template`
```

Comment:
Should this be `gcc` rather than `bcc`?
Also, does the cpan command require the -i switch for installing?